jsonrpc/eth_endpoint.go

```diff
@@ -736,7 +709,7 @@ func (e *Eth) GetTransactionCount(address types.Address, filter BlockNumberOrHas
 	}
 
 	if filter.BlockNumber == nil {
-		header, err = e.getHeaderFromBlockNumberOrHash(&filter)
+		header, err = e.getHeaderFromBlockNumberOrHash(filter)
 		if err != nil {
 			return nil, fmt.Errorf("failed to get header from block hash or block number")
```

Comment:
We ignore error here.

Reply:
This was a special case of the same function that I did not want to remove yet. I will include the error though.